config/mocha.browser.opts

```diff
@@ -0,0 +1,2 @@
+--timeout 20000
```

Comment:
Do we need to include google license as in the file you deleted?

Reply:
Looks like only js/ts files require a license header? Js configuration files under the 'config' directory do have a license header, while all the others don't. But I'm not sure. Your call.

P.S. If we decide to add a header in this mocha config file, mocha version needs an upgrade to support the comments.